projects/ngx-paypal-lib/src/lib/components/paypal.component.ts

```diff
@@ -133,9 +133,11 @@ export class NgxPaypalComponent implements OnChanges, OnDestroy, AfterViewInit {
         this.initializePayPal = true;
 
         if (this.payPalButtonContainerElem) {
-            while (this.payPalButtonContainerElem.nativeElement.firstChild) {
-                this.payPalButtonContainerElem.nativeElement.removeChild(this.payPalButtonContainerElem.nativeElement.firstChild);
-            }
+            try {
+                while (this.payPalButtonContainerElem.nativeElement.firstChild) {
+                    this.payPalButtonContainerElem.nativeElement.removeChild(this.payPalButtonContainerElem.nativeElement.firstChild);
+                }
+            } catch (e) { }
```

Comment:
Could you please add a  'console.error' call here so that it's not silently discarded?

Reply:
Done 👍